src/main/java/mate/academy/hibernate/relations/dao/impl/CountryDaoImpl.java

```diff
@@ -12,11 +15,26 @@ public CountryDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Country add(Country country) {
-        return null;
+        Transaction transaction = null;
+        try (Session session = factory.openSession()) {
+            transaction = session.beginTransaction();
+            session.persist(country);
```

Comment:
Consider using `session.save(country)` instead of `session.persist(country)` if you want to ensure that the entity is immediately saved and the generated identifier is available. `persist` is typically used for entities that are not yet persisted and does not guarantee the immediate generation of an identifier.